src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -517,20 +520,51 @@ def _create_audit_dir(self):
         with open(file_path, "w") as file:
             json.dump(self.env_info_dict, fp=file)
 
-    def collect_commcare_cloud_details(self):
+    def _collect_commcare_cloud_details(self):
         repo = git.Repo(search_parent_directories=True)
         hexsha = repo.head.object.hexsha
         self.env_info_dict["commcare_cloud"] = {"commit": hexsha}
 
-    def collect_commcare_hq_details(self):
+    def _collect_commcare_hq_details(self):
         hexsha = get_deployed_version(self.environment)
         self.env_info_dict["commcare_hq"] = {"commit": hexsha}
 
-    def validate_environment_settings(self):
+    def _validate_environment_settings(self):
         settings_validaton = {"passed": True, "failure_reason": None}
         try:
             self.environment.check()
         except Exception as exception:
             settings_validaton["passed"] = False
-            settings_validaton["failure_reason"] = str(exception)
+            self.env_info_dict["reason"] = str(exception)
         self.env_info_dict["settings_validation"] = settings_validaton
+
+    def _collect_control_machine_os_level_info(self):
+        os_data = {}
+        # Distribution
+        os_distrib = {}
+        with open("/etc/lsb-release", "r") as file:
+            distrib_info = file.read().split("\n")
+            for info in distrib_info:
+                if not info:
+                    continue
+                key = info.split("=")[0]
+                value = info.split("=")[1]
+                os_distrib[key] = value
+        
+        os_data["os_distrib"] = os_distrib
+        
+        # File/Dir stats
+        paths = ["/opt/data"]
+        file_stats = []
+        for path_raw in paths:
+            path = Path(path_raw)
+            stats = {}
+            path_stats = path.stat()
+            stats["path"] = path_raw
```

Comment:
Is this really necessary to log? I get the OS info, but not really the motivation for the `path_stats`?

Reply:
I just think it's not contributing any information as opposed to the OS info, so rather leave it out.

Reply:
It's technically not needed. That said, I did realize this was done wrong. I'm working on an update for this branch